src/DataFixtures/ContentFixtures.php

```diff
@@ -34,10 +34,10 @@ class ContentFixtures extends BaseFixture implements DependentFixtureInterface
     /** @var Config */
     private $config;
 
-    /** @var Areas */
+    /** @var FileLocations */
     private $areas;
```

Comment:
$fileLocations

Reply:
done

Reply:
still not done

Reply:
weird, i could've sworn i did.. It'll e in the next `push`!